code_schemes/s07e01.json

```diff
@@ -3,7 +3,118 @@
   "Name": "S07E01",
   "Version": "0.0.0.1",
   "Codes": [
-
+    {
+      "CodeID": "code-9902881c",
+      "CodeType": "Normal",
+      "DisplayText": "general knowledge on corona",
+      "NumericValue": 1,
+      "StringValue": "general_knowledge_on_corona",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-43674c39",
+      "CodeType": "Normal",
+      "DisplayText": "don't know anything",
+      "NumericValue": 2,
+      "StringValue": "dont_know_anything",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-c90e231f",
+      "CodeType": "Normal",
+      "DisplayText": "(call for) awareness creation",
```

Comment:
is this the new way of displaying L3/L4 codes? probably update theme 12 too?

Reply:
no, this is CSAP where this is no concept of L3/4 codes. I think the brackets are just to help the RAs interpret 'awareness creation'.